2020q2/multiarch-ci.md

```diff
@@ -0,0 +1,34 @@
+## Continuous Integration on !x86 ##
+
+Contact: Edward Tomasz Napierala, <trasz@FreeBSD.org>
+
+For quite a while the [FreeBSD CI infrastructure](https://ci.FreeBSD.org)
+has been running FreeBSD builds and regression tests, making it easy to
+spot regressions.  While CI was building images for all architectures,
+the regression tests were only run on amd64 and i386, which means they
+couldn't detect architecture-specific runtime breakage.  The goal of this
```

Comment:
“architecture-specific runtime breakage” should possible be “architecture-specific run-time breakage on non-x86 architectures”?